### DISCUSSION TITLE
New archive hasn't been checked in 2 days

### DISCUSSION THREAD

**Discussion by garrmark** [renovatebot/renovate#15734] [Request Help]
I created a new archive in github and renovate checked it 2 days ago.  Since then I've made a lot of changes including renaming the default/only branch from main to develop.

I would like renovate to scan it again, is there a way to trigger another scan?

![Screenshot 2022-05-25 123558](https://user-images.githubusercontent.com/2669864/170315345-70833e26-c94b-4f69-8c2c-9ca7b2522aea.jpg)

**Comment by HonkingGoose** [comment]
It looks Renovate doesn't find or recognize any of your package files because Renovate says: `no-package-files`. Before we can help you, we need more information.

Please answer these questions:

- Do you have package files in your repository?
  - If you have package files, please list them.
- Did you "archive" your repository?
